dex/testing/dcrdex/harness.sh

```diff
@@ -174,6 +191,18 @@ if [ $ETH_ON -eq 0 ]; then
 EOF
 fi
 
+if [ $DOGE_ON -eq 0 ]; then
+    cat << EOF >> "./markets.json"
+         },
+        "DOGE_simnet": {
+            "bip44symbol": "doge",
+            "network": "simnet",
+            "maxFeeRate": 20,
```

Comment:
A bit more background on the doge fee rate saga: https://github.com/dogecoin/dogecoin/discussions/2347
Proposed relay and dust values supposedly put into effect in 1.14.4:
https://github.com/dogecoin/dogecoin/pull/2440/files

Not entirely sure what the current right min relay fee is now.

Reply:
What I gather from that plan and the changes implemented in https://github.com/dogecoin/dogecoin/pull/2440/files back in 1.14.4 are:

### fee rates

- `DEFAULT_BLOCK_MIN_TX_FEE` (block inclusion fee limit) is now 0.01 DOGE / kB (1,000 atom / byte)
- `DEFAULT_MIN_RELAY_TX_FEE` (node relay fee limit) is now 0.001 DOGE / kB (100 atom / byte)

Essentially, the mining code (by default, but configurable) is more demanding than relay.  Sensibly, the *recommended* rate is `DEFAULT_FALLBACK_FEE` of 0.01 DOGE/kB or 1,000 atom/B.

So, it's probably a good idea for server's maxFeeRate to be maybe 5-10x of that.  Somewhat arbitrarily, I had testnet server set to 9000.  e.g. https://blockexplorer.one/dogecoin/testnet/tx/0c661154120cc35e778d0130b790de3bbd74707ef7f2203ec4c59e99ffeda6df

There should probably be some checks that maxFeeRate is not set below this limit.  Thoughts about that?

### dust

- The **hard** dust limit is 0.001 DOGE - "outputs under this value are invalid and rejected."
- The **soft** dust limit is 0.01 DOGE - "sending a transaction with outputs under this value, are required to add 0.01 DOGE for each such output, or else will be considered to have too low fee and be rejected."

The *recommended* dust limit (discard limit) is 1.00 DOGE.

Our dust checks should be sufficient since the function takes the min relay fee as an input: `func IsDust(txOut *wire.TxOut, minRelayTxFee uint64) bool`
In calling this function, we actually pass the targeted fee rate for the transaction being authored, so _as long as we check that that rate is at least the min relay fee_ (we aren't yet), that is adequate.

In summary, it sounds like there's use for a `dex/networks/doge.MinRelayFee` variable for both server and client for the above reasons.  Client can use it internally to `client/asset` in it's `targetFeeRateWithFallback` and/or `feeRateWithFallback` methods, while server can use it when setting up the backed asset (or wherever) to ensure maxFeeRate is at least that amount.

Reply:
Anyway, I guess we're fine on server side for as long as the operator picks a high enough maxFeeRate.

But I think we want to change client's (fallback) `defaultFee = 500000` to `50_000` or less, and the `defaultFeeRateLimit` to `200_000` or less

Reply:
Yup, we definitely have an issue with `IsDust`

Reply:
Long past time for me to change gears, but I arrived here: https://github.com/chappjc/dcrdex/commit/638ded8d2c259e7ec7a46594d80c8d37e460578a